[2024-07-25T13:17:32.740+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: basic_logging_dag.load_data scheduled__2023-04-01T00:00:00+00:00 [queued]>
[2024-07-25T13:17:33.014+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: basic_logging_dag.load_data scheduled__2023-04-01T00:00:00+00:00 [queued]>
[2024-07-25T13:17:33.023+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-07-25T13:17:33.488+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-04-01 00:00:00+00:00
[2024-07-25T13:17:33.580+0000] {standard_task_runner.py:57} INFO - Started process 342 to run task
[2024-07-25T13:17:33.641+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'basic_logging_dag', 'load_data', 'scheduled__2023-04-01T00:00:00+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/basic_logging_email/basic_logging_dag.py', '--cfg-path', '/tmp/tmp5pz8y2d1']
[2024-07-25T13:17:33.672+0000] {standard_task_runner.py:85} INFO - Job 33: Subtask load_data
[2024-07-25T13:17:34.215+0000] {task_command.py:416} INFO - Running <TaskInstance: basic_logging_dag.load_data scheduled__2023-04-01T00:00:00+00:00 [running]> on host b7334bec8397
[2024-07-25T13:17:36.477+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='alexioaaron75@gmail.com' AIRFLOW_CTX_DAG_OWNER='Ajay' AIRFLOW_CTX_DAG_ID='basic_logging_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-04-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-04-01T00:00:00+00:00'
[2024-07-25T13:17:36.483+0000] {basic_logging_dag.py:38} INFO - Finally load data
[2024-07-25T13:17:36.500+0000] {basic_logging_dag.py:39} ERROR - Oh, where is the data?
[2024-07-25T13:17:36.520+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-07-25T13:17:36.637+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=basic_logging_dag, task_id=load_data, execution_date=20230401T000000, start_date=20240725T131732, end_date=20240725T131736
[2024-07-25T13:17:36.990+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-07-25T13:17:37.550+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
